### PR TITLE
rename 'underrepresented students of color' to 'underrepresented mino…

### PR DESCRIPTION
**Pull request by tanyaparker**
…rities'

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/privacy.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Date of Last Revision: November 17, 2016
 
 
 ## About Code.org
-Code.org® is a nonprofit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented students of color.
+Code.org® is a nonprofit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented minorities.
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/privacy.md:11]: @poorvasingal Is it possible to update our privacy policy and policy notice?

Comment (ashercodeorg) [reply pegasus/sites.v3/code.org/public/privacy.md:11]: LGTM, we are allowed to make minor changes to the policy (as this is) without notifying our users. However, I would confirm the change with Hadi.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/privacy.md:11]: Great. Request is from Hadi/Jon. I'll confirm he's ok it affects privacy.

File: pegasus/sites.v3/code.org/public/privacy-apr-2016.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Planning to update privacy policy: Aug 24, 2016
 
 
 ## About Code.org
-Code.org® is a non-profit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented students of color.
+Code.org® is a non-profit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented minorities.
```
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/public/privacy-apr-2016.md:15]: Previous privacy policies should NOT be changed, they should be maintained for historical record.

File: pegasus/sites.v3/code.org/public/privacy-notice.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ title: Privacy Notice
 Date of Last Revision: August 24, 2016
```
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/public/privacy-notice.md:7]: Does the date of last revision get updated automatically or does it need to be updated here?

Comment (ashercodeorg) [reply pegasus/sites.v3/code.org/public/privacy-notice.md:7]: And what is the difference between `privacy-notice.md` and `privacy.md`?

File: pegasus/sites.v3/code.org/public/star-wars-announcement.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ And sign up to participate in December's Hour of Code event: [https://hourofcode
 
 **About Code.org**
 
-Code.org is a 501c3 public non-profit dedicated to expanding access to computer science and increasing participation by women and underrepresented students of color. Its vision is that every student in every school should have the opportunity to learn computer science. After launching in 2013, Code.org organized the Hour of Code campaign – which has introduced over 100 million students to computer science to date – and partnered with 70 public school districts nationwide to expand computer science programs. Code.org is supported by philanthropic donations from corporations, foundations and generous individuals, including Ballmer Family Giving, Google, Infosys Foundation USA, Microsoft, Omidyar Network and others.
```
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/public/star-wars-announcement.md:66]: Does it make sense to keep or update prior announcements?

Probably makes sense to confirm each of these changes individually.

**Comment by tanyaparker** [comment]
I only searched code.org but actually I'll expand my search to dashboard and our other pegasus sites.

File: pegasus/sites.v3/code.org/public/privacy.md (hunk 1 above)
Comment (poorvasingal) [reply pegasus/sites.v3/code.org/public/privacy.md:11]: Yep, what Asher said.

**Pull request closed without merging**